content/_help/verify-your-identity/verify-your-identity-in-person._fr.md

```diff
@@ -0,0 +1,135 @@
+   WASHINGTON, D. C. 20016-9997
+4. **WASHINGTON — Post Office™**
+   900 BRENTWOOD RD NE
+   WASHINGTON, D. C. 20018-9997
+5. **ARLINGTON — Post Office™**
+   3118 WASHINGTON BLVD
+   ARLINGTON, VA 22201-9998
+6. **ASHBURN — Post Office™**
+   44715 PRENTICE DR
+   DULLES, VA 20101-9996
+7. **MERRIFIELD — Post Office™**
+   8409 LEE HWY
+   MERRIFIELD, VA 22116-9998
+
+### Vous pouvez parler à n'importe quel associé du magasin pour vérifier votre identité
+
+* Faites la queue au bureau de poste. L'employé scannera votre code-barres et examinera vos documents.
+* Une fois que l'employé aura examiné vos documents, vous avez terminé au bureau de poste. Cet employé ne recevra pas vos résultats.
+
+### Nous vous enverrons un courriel avec les prochaines étapes
+
+Nous vous enverrons un courriel pour vous informer de la réussite ou de l'échec de la vérification de votre identité dans les 24 heures suivant votre visite au bureau de poste. 
+
+Si la vérification de votre identité n'a pas abouti, vous pouvez :
+
+* Essayer à nouveau de vérifier votre identité en ligne.
+* Contacter le centre d'assistance de l'agence partenaire pour obtenir de l'aide.
+
+Si vous ne recevez pas de courriel de notre part, il y a quelques étapes que vous pouvez suivre :
+
+* Vérifiez dans le dossier de pourriels ou de courriers indésirables de votre courriel s'il y a un courriel de [no-reply@login.gov](mailto:no-reply@login.gov).
+* Si vous n'avez pas eu de nouvelles après 24 heures de votre visite au bureau de poste, veuillez contacter le service clientèle de Login.gov sur le site login.gov/contact.
+
+# Résolution des problèmes liés à la vérification de votre identité en personne
+
+Si vous avez rencontré un problème en essayant de vérifier votre identité en personne, nous aimerions vous aider à le résoudre. Voici quelques conseils pour résoudre le problème. 
+
+## Ce qu’il faut faire si vous avez reçu un courriel, mais que vous n’avez pas essayé de vérifier votre identité en personne
+
+Si vous avez reçu un courriel vous informant que vous êtes prêt à vérifier votre identité ou que votre identité n’a pas pu être vérifiée en personne, et que vous n’avez pas tenté cette action: 
+
+[Connectez-vous](https://secure.login.gov/fr/) et [changez votre mot de passe immédiatement](/fr/help/manage-your-account/change-your-password/). Si vous le souhaitez, vous pouvez également [supprimer votre compte](/fr/help/manage-your-account/delete-your-account/).
+
+## Ce qu’il faut faire si vous n’avez pas pu vérifier votre identité en personne
+
+Si vous ne parvenez pas à vérifier votre identité en personne, vous pouvez réessayer de [vérifier votre identité en ligne](/fr/help/verify-your-identity/how-to-verify-your-identity/) ou contacter l’agence à laquelle vous essayez d’accéder [pour obtenir de l’aide sur des questions spécifiques](/fr/help/specific-agencies/overview/) relatives la vérification de votre identité.
+
+### Conseils pour vérifier votre identité en personne: 
```

Comment:
This isn't a heading in the other languages. Do we mean for it to be in the French version?

```suggestion
Conseils pour vérifier votre identité en personne: 
```

Reply:
Haha I just saw this too! Should be fixed in the latest commit.